easybuild/easyconfigs/r/rstudio/rstudio-1.3.959-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
+version = '1.3.959'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+
+homepage = 'https://www.rstudio.com/'
+description = """This RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R."""
```

Comment:
@branfosj Why have you removed the information on how to start the server?

Reply:
I based this off our local copy and that does not have the information on how to start the server. I've just updated to put the information back in.

Reply:
@branfosj thanks.